doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -317,3 +317,14 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
        running. It can only be powered off from inside the VM itself.
 
        By default, this option is not enabled.
+
+   * - :kbd:`--pm_notify_channel`
```

Comment:
```suggestion
   * - :kbd:`--pm_notify_channel <channel>`
```

Reply:
Geoff, thanks a lot for the suggestions. I will update the PR accordingly.